piptools/sync.py

```diff
@@ -14,8 +14,14 @@
     'pip-tools',
     'pip-review',
     'pkg-resources',
+
+    # stdlib and dev packages
     'setuptools',
     'wheel',
+    'distribute',
+    'python',
+    'wsgiref',
+    'argparse',
```

Comment:
I assume there would be some way to detect this list automatically, but it would likely require to import it then?

Reply:
Yes. Those packages could be found here
https://github.com/pypa/pip/blob/b38e835ab2f65bea6d9e712f459b3e8e01abb0ea/src/pip/_internal/utils/compat.py#L206

and there: https://github.com/pypa/pip/blob/0d9c05ec32bc420547d846a780507d44acd9a4a6/src/pip/_internal/commands/freeze.py#L11

But there would be some compatibility and circular imports issues, though.

Reply:
I was wrong. Check the last commit.

Reply:
Awesome!